src/util/types.ts

```diff
@@ -63,12 +63,14 @@ export interface UIOrder {
     filled: BigNumber;
     price: BigNumber;
     status: OrderStatus;
+    emptySize: BigNumber;
```

Comment:
This works but having a "denormalized" property might be error-prone and hurt maintainability, since this opens the door for having inconsistent states (where `emptySize + filled !== size`). Since this value can be computed from the others, I think it's best to just compute it when necessary. If this turns out to happen a lot, we can convert this type to a class with a getter method for `emptySize`.

And while we are on it, I'm not sure about the name `emptySize`. Maybe something like `available`?